common/src/barretenberg_structures.rs

```diff
@@ -91,6 +106,21 @@ impl Constraint {
 
         buffer
     }
+
+    fn set_linear_term(&mut self, coef: Scalar, witness: i32) {
+        if self.a == 0 || self.a == witness {
+            self.a = witness;
+            self.ql = coef;
```

Comment:
spellcheck on `coef`

Reply:
?

Reply:
This is throwing warnings in CI for the spellchecker.

Reply:
Done